src/librustc/cfg/construct.rs

```diff
@@ -150,16 +150,14 @@ impl<'a, 'tcx> CFGBuilder<'a, 'tcx> {
         }
     }
 
-    fn pats_all<'b, I: Iterator<Item=&'b P<hir::Pat>>>(
-        &mut self,
-        pats: I,
-        pred: CFGIndex
-    ) -> CFGIndex {
+    fn pats_all<'b, 'h: 'b, I: Iterator<Item=&'b P<'h, hir::Pat<'h>>>>(&mut self,
+                                          pats: I,
+                                          pred: CFGIndex) -> CFGIndex {
```

Comment:
Intrigued by this change, the previous formatting was correct.

Reply:
There are a few of these changes were formatting is reverted to the formatting at the time I created the commit. Totally could use some `rustfmt` help.

Reply:
Could you clean the indenting back to what it was here?